volplugin/volplugin.go

```diff
@@ -166,50 +166,56 @@ func (dc *DaemonConfig) getGlobal() {
 
 func (dc *DaemonConfig) watchGlobal() error {
 	for {
-		time.Sleep(1 * time.Minute)
+		time.Sleep(1 * time.Second)
 		dc.getGlobal()
 	}
 }
 
 func (dc *DaemonConfig) updateMounts() error {
-	cd := ceph.NewDriver()
-	mounts, err := cd.Mounted(dc.Global.Timeout)
-	if err != nil {
-		return err
-	}
+	for driverName := range backend.Drivers {
+		cd, err := backend.NewDriver(driverName, dc.Global.MountPath)
```

Comment:
i don't think so. Policies are uniquely named, namespacing volumes. If 
the drivers manage themselves this way, there will be no conflicts.

On 8 Mar 2016, at 15:49, Madhav Puri wrote:

>> 		dc.getGlobal()
>> 	}
>> }
>>
>> func (dc *DaemonConfig) updateMounts() error {
>> -	cd := ceph.NewDriver()
>> -	mounts, err := cd.Mounted(dc.Global.Timeout)
>> -	if err != nil {
>> -		return err
>> -	}
>> +	for driverName := range backend.Drivers {
>> +		cd, err := backend.NewDriver(driverName, dc.Global.MountPath)
>
> @erikh I am just wondering if having a global mount-path work across 
> drivers. Won't separate drivers step over each other, if they try to 
> manage the same mount path?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/171/files#r55450037